### PR TITLE
SI-46 Endpoint for Connect Service

### PR DESCRIPTION
**Pull request by jeanhall**

### CODE REVIEW

File: Zone5/Views/ThirdPartyConnectionsView.swift [hunk 1]
```diff
@@ -35,16 +36,14 @@ public class ThirdPartyConnectionsView: APIView {
...
+	@available(*, deprecated, message: "Use `pairThirdPartyConnection(type:redirect:completion)` instead.")
 	public func initializeThirdPartyConnection(type: UserConnectionType, completion: @escaping Zone5.ResultHandler<ThirdPartyInitializeResponse>) -> PendingRequest? {
 		let endpoint = Endpoints.initializePairing.replacingTokens(["connectionType": type.connectionName])
 		return post(endpoint, body: EmptyBody(), with: completion)
 	}
-	
-	/// Set an access token for the current user for the given 3rd party type
-	/// - Parameters
+
+	/// @available(*, deprecated, message: "Use `pairThirdPartyConnection(type:redirect:completion)` instead.")
```
Comment (jellybeansoup) [inline Zone5/Views/ThirdPartyConnectionsView.swift]: Is this supposed to be uncommented (and thus annotating `setThirdPartyToken(type:parameters:completion:)` below), or did you not intend to leave it in?

**Review by jellybeansoup — APPROVED** [review]
Should probably do something with that `@available` annotation you've commented out, but otherwise these changes look good.

File: Zone5/Views/ThirdPartyConnectionsView.swift (hunk 1 above)
Comment (jeanhall) [reply Zone5/Views/ThirdPartyConnectionsView.swift]: ha - good catch. I was definitely intending to uncomment that. Thanks!

Comment (jeanhall) [reply Zone5/Views/ThirdPartyConnectionsView.swift]: fixed

**Pull request merged**